perceval/backends/core/confluence.py

```diff
@@ -332,6 +352,8 @@ def contents(self, from_date=DEFAULT_DATETIME,
         :param limit: maximum number of contents to fetch per request
         """
         resource = self.RCONTENTS + '/' + self.MSEARCH
+        if self.add_ancestors:
```

Comment:
I would move this code to another method (e.g., `ancestors`) to improve the readbility of the code. What do you think ?

Reply:
As per comments above, if we want to always enable ancestors enrichment, then I'd simply change line 354 to `resource = self.RCONTENTS + '/' + self.MSEARCH + '?' + self.PEXPAND '=' + self.PANCESTORS` and remove the other part of the change. WDYT?

Reply:
I would put it in a different method to:
- be sure that the paginated ancestors are returned (I'm not sure whether the code you proposed that care of this)
- simplify the testing